xbmc/music/Song.h

```diff
@@ -195,6 +206,7 @@ class CSong final : public ISerializable
   std::string strRecordLabel; // Record label from tag for album processing by CMusicInfoScanner::FileItemsToAlbums
   std::string strAlbumType; // (Musicbrainz release type) album type from tag for album processing by CMusicInfoScanner::FileItemsToAlbums
   std::string songVideoURL; // url to song video
+  std::map<int, ChapterDetails> m_chapters; // map of chapter names and start and end times
```

Comment:
Well, as you have the "using ChapterMarks" in line 52, you should use it here: => `ChapterMarks m_chapters;`